auth-api/tests/unit/services/test_org.py

```diff
@@ -403,6 +405,54 @@ def test_create_product_multiple_subscription(session, keycloak_mock, monkeypatc
                 if prod.get('code') == TestOrgProductsInfo.org_products2['subscriptions'][1]['productCode'])
 
 
+def test_create_product_subscription_staff(session, keycloak_mock, monkeypatch):
+    """Assert that updating product subscription works for staff."""
+    user = factory_user_model(TestUserInfo.user_test)
+    patch_token_info({'sub': user.keycloak_guid}, monkeypatch)
+    org = OrgService.create_org(TestOrgInfo.org1, user_id=user.id)
+
+    # Clearing the event listeners here, because we can't change the type_code.
+    clear_event_listeners(OrgModel)
+    org_db = OrgModel.find_by_id(org._model.id)
+    org_db.type_code = OrgType.STAFF.value
+    org_db.save()
+    event.listen(OrgModel, 'before_update', receive_before_update, raw=True)
+    event.listen(OrgModel, 'before_insert', receive_before_insert)
+
+    subscriptions = ProductService.create_product_subscription(org._model.id,
+                                                               TestOrgProductsInfo.org_products2,
+                                                               skip_auth=True)
+
+    assert next(prod for prod in subscriptions
+                if prod.get('code') == TestOrgProductsInfo.org_products2['subscriptions'][0]['productCode'])
+    assert next(prod for prod in subscriptions
+                if prod.get('code') == TestOrgProductsInfo.org_products2['subscriptions'][1]['productCode'])
+
+
+def test_create_product_subscription_sbc_staff(session, keycloak_mock, monkeypatch):
```

Comment:
to prevent the duplicate code in here can you add this test case to the above one and pass in the org type as a param?

Reply:
Yup